alisa.go

```diff
@@ -214,7 +194,7 @@ func (ali *alisa) stop(taskID string) (bool, error) {
 }
 
 func (ali *alisa) requetAndParseResponse(params map[string]string) (*json.RawMessage, error) {
-	rspBuf, err := ali.pop.request(params, ali.popURL, ali.popSecret)
+	rspBuf, err := ali.pop.request(params, ali.POPURL, ali.POPAccessKey)
```

Comment:
I would suggest renaming `popSecret` to `POPAccessSecret`.

Reply:
Done.